src/api/__test__/api-period.test.js

```diff
@@ -434,15 +479,6 @@ describe('ttrack API',() => {
                 const result = await query(`SELECT * from periods WHERE per_id = ${period.per_id}`);
                 expect(result.rows.length).toBe(0);
             });
-
-            it("should not throw any errors if period is unknown", async () => {
```

Comment:
why did you remove that?
Delete on a Period that is not there should be idempotent, therefore always returning 204